topics/assembly/tutorials/ERGA-post-assembly-QC/tutorial.md

```diff
@@ -656,7 +693,7 @@ Bandage is a tool to visualise *de novo* assembly graphs with connections. ({% c
 > <hands-on-title> Generate assembly graph </hands-on-title>
 >
 > 1. {% tool [gfastats](toolshed.g2.bx.psu.edu/repos/bgruening/gfastats/gfastats/1.3.6+galaxy0) %} with the following parameters:
->    - {% icon param-file %} *"Input file"*: `output` (Input dataset)
+>    - {% icon param-file %} *"Input file"*: `CReformitis_assembly` (Input dataset)
```

Comment:
Does the R needs to be capital?

Reply:
There are still some discordances in the names that can be improved overall, but I think it runs well enough for the training for now